<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BankerMainWindow</class>
 <widget class="QWidget" name="BankerMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>541</width>
    <height>381</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>541</width>
    <height>381</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>541</width>
    <height>381</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Banker Main Window</string>
  </property>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>40</y>
     <width>541</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>50</y>
     <width>541</width>
     <height>331</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QLabel" name="WelcomeLabel">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>541</width>
       <height>111</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>24</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Welcome,</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="Text2">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>130</y>
       <width>541</width>
       <height>111</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>28</pointsize>
      </font>
     </property>
     <property name="text">
      <string>What would you like to do?</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>260</y>
       <width>75</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Log out</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_3">
    <widget class="QComboBox" name="senderBox">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>110</y>
       <width>121</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="receiverBox">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>150</y>
       <width>121</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>541</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>26</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Transfer</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>110</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>From:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>150</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>To:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="amountBox">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>190</y>
       <width>121</width>
       <height>22</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>0.00</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>190</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Amount:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>270</y>
       <width>81</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Submit</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>190</y>
       <width>20</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Â£</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QTableWidget" name="customersTable">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>521</width>
       <height>261</height>
      </rect>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="textElideMode">
      <enum>Qt::ElideMiddle</enum>
     </property>
     <attribute name="horizontalHeaderVisible">
      <bool>true</bool>
     </attribute>
     <attribute name="horizontalHeaderCascadingSectionResizes">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderMinimumSectionSize">
      <number>39</number>
     </attribute>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>165</number>
     </attribute>
     <attribute name="horizontalHeaderHighlightSections">
      <bool>true</bool>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <column>
      <property name="text">
       <string>Name</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Balance</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>User type</string>
      </property>
     </column>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>541</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>26</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Customers</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="homeButton">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Home</string>
   </property>
  </widget>
  <widget class="QPushButton" name="transferButton">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>0</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Transfer</string>
   </property>
  </widget>
  <widget class="QPushButton" name="customersButton">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>0</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Customers</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
